### PR TITLE
[Doc] Update the Cumulocity public doc about the Actility application EUI moving to Device registration page from connectivity setting page

### PR DESCRIPTION
**Pull request by ashhSAG93**

### CODE REVIEW

**Review by miryalarahul — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 1]
```diff
@@ -9,15 +9,14 @@ Before using LoRa devices with {{< product-c8y-iot >}}, you need to configure yo
 
 ### <a name="create-new-credentials">Creating new account credentials</a>
 
-If you select **Connectivity** for the first time, you are asked to provide the Actility ThingPark URL, credentials and application EUI which is used for LoRa device provisioning.
+If you select **Connectivity** for the first time, you are asked to provide the Actility ThingPark URL and credentials which is used for LoRa device provisioning.
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:12]: which is used > which are used

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 2]
```diff
@@ -31,7 +30,7 @@ Click **Save connectivity settings**. If you have entered the correct informatio
 
 In order to update your connectivity settings, go to the Administration and click **Update connectivity settings** in the **Settings** menu.
 
-Enter your Actility ThingPark URL, profile ID, username, password and application EUI as requested. For details on these information see [Creating new account credentials](#create-new-credentials).
+Enter your Actility ThingPark URL, profile ID, username and password as requested. For details on these information see [Creating new account credentials](#create-new-credentials).
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:33]: @MWindel Shouldn´t there be a comma after "username"?

**Comment by BeateRixen** [comment]
The image lora-admin-settings.png does not match our guidelines for screenshots. The browser settings should be 1440 x 900 pixels when taking screenshots, especially when showing the full screen, please see also https://docs.google.com/document/d/1geBLxk35ZIiS8C2Uo15H-14oneuEEKRf0eerHqdm3fw/edit#heading=h.bqw50lhopz2p.

**Comment by ashhSAG93** [comment]
> The image lora-admin-settings.png does not match our guidelines for screenshots. The browser settings should be 1440 x 900 pixels when taking screenshots, especially when showing the full screen, please see also https://docs.google.com/document/d/1geBLxk35ZIiS8C2Uo15H-14oneuEEKRf0eerHqdm3fw/edit#heading=h.bqw50lhopz2p.

I have changed it to be 1440x900

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md (hunk 2 above)
Comment (ashhSAG93) [reply content/protocol-integration/lora-actility-bundle/configure-credentials.md:33]: @MWindel @BeateRixen Should I make this change

Comment (ashhSAG93) [reply content/protocol-integration/lora-actility-bundle/configure-credentials.md]: Its done

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**